libparsec/crates/protocol/src/version.rs

```diff
@@ -14,6 +14,7 @@ use libparsec_types::prelude::*;
 // - v4.1 (Parsec 3.2+):
 //   * Add `ShamirRecovery` variants to `invite_list` and `invite_info`
 //   * Add `invite_new_shamir_recovery` command
+// v5 (Parsec 3.3+): TODO
```

Comment:
Should create an issue to track the fact we need to update this before 3.3 release

Reply:
ad797d8d07527399bca8277bc2b393a61fd0aff5